torch/export/_safeguard.py

```diff
@@ -0,0 +1,15 @@
+import torch
+from torch.overrides import TorchFunctionMode
+
+
+class GradModeUnsupportedSafeguard(TorchFunctionMode):
```

Comment:
Rename this to: "DetectUnsafeGlobalStateOpsInExportMode"?

Reply:
Looks like you try to generalize the functionality on global state ops, instead of just grad mode state. Do you intend to do so? `InExportMode` may be redundant, as the file path indicates that.